src/test/unit_tests/test_lifecycle.js

```diff
@@ -291,21 +291,24 @@ mocha.describe('lifecycle', () => {
         });
     });
 
-    mocha.describe('bucket-lifecycle-version', function() {
+    mocha.describe('bucket-lifecycle-version', async function() {
         this.timeout(60000);
         const version_bucket = 'test-version-bucket';
-        mocha.after(async function() {
-            //TODO Delete bucket
-            //await rpc_client.bucket.delete_bucket({ name: version_bucket });
+
+        await rpc_client.bucket.create_bucket({ name: version_bucket});
+        rpc_client.bucket.update_bucket({
+            name: version_bucket,
+            versioning: 'ENABLED'
         });
 
-        async function create_mock_version(version_key, bucket, age, version_count) {
-            await rpc_client.bucket.create_bucket({ name: bucket });
-            rpc_client.bucket.update_bucket({
-                name: bucket,
-                versioning: 'ENABLED'
-            });
+        mocha.after(async function() {
+            for (let i = 0; i < 2; ++i) {
+                await rpc_client.object.delete_object({ bucket: version_bucket, key: `test-lifecycle-version${i}`});
+            }
+            await rpc_client.bucket.delete_bucket({ name: version_bucket });
+        });
 
+        async function create_mock_version(bucket, version_key, age, version_count) {
```

Comment:
I have made some changes in this function in #8888

Reply:
there are a few changes in both test and lifecycle.js, I will merge it after merging PR [8888](https://github.com/noobaa/noobaa-core/pull/8888), WDYT?